<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCalc 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/img/logo-16x16.png</normaloff>:/icons/img/logo-16x16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="horizontalSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="dataLabel">
           <property name="text">
            <string>Pattern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DataTableView" name="dataTable">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="costMatrixLabel">
           <property name="text">
            <string>Cost matrix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CostMatrixTableView" name="costMatrixTable">
           <property name="selectionMode">
            <enum>QAbstractItemView::ContiguousSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="movementsLabel">
           <property name="text">
            <string>Transfers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="OutputTableWidget" name="movementsTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="output">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier New'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="separator"/>
    <addaction name="importDataAction"/>
    <addaction name="exportDataAction"/>
    <addaction name="separator"/>
    <addaction name="importCostMatrixAction"/>
    <addaction name="exportCostMatrixAction"/>
    <addaction name="separator"/>
    <addaction name="exportProblemToGnuMathProgAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="calculateAction"/>
    <addaction name="separator"/>
    <addaction name="symmetricEditModeAction"/>
    <addaction name="loadDistanceMatrixAction"/>
    <addaction name="loadUMatrixAction"/>
    <addaction name="separator"/>
    <addaction name="clearOutputAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="separator"/>
   <addaction name="loadDistanceMatrixAction"/>
   <addaction name="loadUMatrixAction"/>
   <addaction name="symmetricEditModeAction"/>
   <addaction name="separator"/>
   <addaction name="calculateAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadDistanceMatrixAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/distance-matrix.png</normaloff>:/icons/img/distance-matrix.png</iconset>
   </property>
   <property name="text">
    <string>Load distance matrix</string>
   </property>
  </action>
  <action name="calculateAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/media-playback-start.png</normaloff>:/icons/img/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
   <property name="toolTip">
    <string>Calculate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="symmetricEditModeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/symmetric-mode.png</normaloff>:/icons/img/symmetric-mode.png</iconset>
   </property>
   <property name="text">
    <string>Symmetric mode</string>
   </property>
  </action>
  <action name="loadUMatrixAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/u-matrix.png</normaloff>:/icons/img/u-matrix.png</iconset>
   </property>
   <property name="text">
    <string>Load U matrix</string>
   </property>
  </action>
  <action name="clearOutputAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/edit-clear.png</normaloff>:/icons/img/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear output</string>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/document-new.png</normaloff>:/icons/img/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/system-log-out.png</normaloff>:/icons/img/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="importCostMatrixAction">
   <property name="text">
    <string>Import cost matrix...</string>
   </property>
  </action>
  <action name="exportCostMatrixAction">
   <property name="text">
    <string>Export cost matrix...</string>
   </property>
  </action>
  <action name="importDataAction">
   <property name="text">
    <string>Import pattern...</string>
   </property>
   <property name="toolTip">
    <string>Import pattern</string>
   </property>
  </action>
  <action name="exportDataAction">
   <property name="text">
    <string>Export pattern...</string>
   </property>
   <property name="toolTip">
    <string>Export pattern</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/img/dialog-information.png</normaloff>:/icons/img/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="exportProblemToGnuMathProgAction">
   <property name="text">
    <string>Export problem to GNU MathProg...</string>
   </property>
   <property name="toolTip">
    <string>Export problem</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DataTableView</class>
   <extends>QTableView</extends>
   <header>datatableview.h</header>
  </customwidget>
  <customwidget>
   <class>CostMatrixTableView</class>
   <extends>QTableView</extends>
   <header>costmatrixtableview.h</header>
  </customwidget>
  <customwidget>
   <class>OutputTableWidget</class>
   <extends>QTableWidget</extends>
   <header>outputtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
